iiab/iiab-admin-console
# Create menus for IIAB roles, kiwix, rachel, and add kiwix statistics

Testing:

1. start with a dec 23 image
2. git pull iiab
3. ./iiab-install --reinstall
4. Clean install of master on iiab and iiab-admin-console.
5. on iiab-admin-console, do a pull, and ./install
6. Verify function of admin console and home page.
7. Remove all menu.json entries except first ald last
8. pull map2menu branch from georgejhunt/iiab/ 
9. cp the local_med_vars.yml to /etc/iiab
10. iiab-install --reinstall (had to turn off cups, and samba because of errors -- also did apt-get update)
11. pull map2menus branch from georgejhunt/iiab-admin-conosole
12. ./install
13. Results: menu now has kalite,text_zim,mediawiki,elgg,lokole,nextcloud,wordpess,kolibri,moodle,sugarizer
14. Install 5 kiwix items (small total 180mb)
15. run iiab-update-menus in /usr/bin/
16. 5 menu items added -- 2 were missing logos phet, and archwiki
17. Install 4 oer2go items (less than 100Mb)
18. run iiab-update-menus
19. Only one of the 4 showed up on home page (still some bugs)

I have installed this on top of 0.3-devel as well as on top of master
